src/transaction/transaction_master_group_complete_manager.cpp

```diff
@@ -159,14 +181,27 @@ namespace cubtx
 	return;
       }
 
+    while (!is_latest_closed_group_prepared_for_complete ())
+      {
+	/* It happens rare. */
+	thread_sleep (10);
+      }
+
+    if (!starts_latest_closed_group_complete ())
+      {
+	/* Already started by others. */
+	return;
+      }
+
     tx_group &closed_group = get_latest_closed_group ();
 
     /* TODO - consider parameter for MVCC complete here. */
     /* Add group commit log record and wakeup  log flush daemon. */
-    log_append_group_complete (thread_p, tdes, m_latest_closed_group_start_stream_position, closed_group,
-			       &closed_group_commit_lsa, &has_postpone);
-
-    log_wakeup_log_flush_daemon ();
+    /* TODO - fix append group complete */
```

Comment:
what is the fix needed here?

Reply:
The purpose was to avoid future merge issue. In fact, it is the next line of code (commented).
I will remove it for now to be clear for reviewers and hope to not forget at future merge.